packages/react-native/React/Views/ScrollView/RCTScrollView.m

```diff
@@ -1055,6 +1055,25 @@ -(type)getter                                           \
 RCT_SET_AND_PRESERVE_OFFSET(setShowsVerticalScrollIndicator, showsVerticalScrollIndicator, BOOL)
 RCT_SET_AND_PRESERVE_OFFSET(setZoomScale, zoomScale, CGFloat);
 
+- (void)setScrollIndicatorInsets:(UIEdgeInsets)value
+{
+  CGPoint contentOffset = _scrollView.contentOffset;
```

Comment:
@cipolleschi I just restore the old logic, https://github.com/facebook/react-native/blob/a4430b52e5ed74b71a1b8fc669e8d7e68ab453e8/packages/react-native/React/Views/ScrollView/RCTScrollView.m#L1037 , maybe it's because setting indicator insets may affects the contentOffset.

Reply:
it should not. There should not be side effects, and even if there are, they should be contained in the UIScrollView. Can we test what happen if we remove those extra lines?

Reply:
I test it, seems not affects contentoffset, I removed it.